### ISSUE TITLE
[Generic UART] 123; Configure UART not used by default

### ISSUE THREAD

**Issue by ralcaide** [Sinapse-Energia/AP-Sinapse#106]
[Config UART](https://github.com/Sinapse-Energia/AP-Sinapse/wiki/MQTT-Sinapse-API#configure-uart)

**Comment by sinapseenergia** [comment]
Note: 
Depending on HW configuration or model, some UARTs can be occuped by an already connected peripherial

In this case, after configuration request over already used UART, "ERROR" message should be send (Refered to Error Messages Table (This will be upload asap))

**Comment by fpgamcu** [comment]
For now used 999; message if uart is busy or error appears.

**Comment by ralcaide** [comment]
@fpgamcu , is this command developed outside of the engine? why?

**Comment by fpgamcu** [comment]
While processing this command we need to set return message if fail or don't return anything if ok. For now we don't have possibility to manipulate return message from the engine in this way.
But now we can define in context text variable and use it for returning result. If it is ok to have message also when command succeed - I can change the code and move command to engine.

**Comment by ralcaide** [comment]
Yes, please, insert it in the Engine returning always a result. Thank you
